package.json

```diff
@@ -0,0 +1,57 @@
+{
+  "name": "commercetools-sunrise-data",
+  "version": "1.0.0",
+  "description": "[![Build Status](https://travis-ci.org/commercetools/commercetools-sunrise-data.svg?branch=master)](https://travis-ci.org/commercetools/commercetools-sunrise-data)",
+  "main": "index.js",
+  "scripts": {
+    "set:authUrl": "npm config set commercetools-sunrise-data:authUrl",
+    "set:apiUrl": "npm config set commercetools-sunrise-data:apiUrl",
+    "set:projectKey": "npm config set commercetools-sunrise-data:project",
+    "set:clientId": "npm config set commercetools-sunrise-data:client_id",
+    "set:clientSecret": "npm config set commercetools-sunrise-data:client_secret",
+    "get:config": "echo \"authUrl\": && npm config get commercetools-sunrise-data:authUrl && echo \"apiUrl\": && npm config get commercetools-sunrise-data:apiUrl && echo \"projectKey\": && npm config get commercetools-sunrise-data:project && echo \"clientId\": && npm config get commercetools-sunrise-data:client_id && echo \"clientSecret\": && npm config get commercetools-sunrise-data:client_secret",
+    "build": "rimraf dist/ && babel ./**/*.js --out-dir dist/ --ignore ./node_modules,./.babelrc,./package.json,./npm-debug.log --copy-files",
```

Comment:
just out of curiosity: why do you need to force-delete the build directory?   babel etc. should not require that by default and I haven't needed it yet either.

Reply:
During development, it could happen that a file is deleted, but not cleaning up the dist the file will remain in the dist directory, which could potentially cause issues in the future.

Reply:
understand, it's just "potentially":  But since the target users are not always experienced node devs it's probably a good idea to stay on the safe side.